<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Points Layer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxPointsLayer"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Point Name Column</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parcels Layer</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="comboBoxParcelsLayer"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Plot Name Column</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Export to ...</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLineEdit" name="lineEditExportTo"/>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="pushButtonExport">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditPointNameColumn"/>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="lineEditPlotNameColumn"/>
   </item>
   <item row="13" column="1">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
